Gravity/Gravity/DAL/RSAPI/RsapiDao.Get.cs

```diff
@@ -86,8 +88,63 @@ private T GetHydratedDTO<T>(RDO objectRdo, ObjectFieldsDepthLevel depthLevel)
 			where T : BaseDto, new()
 		{
 			T dto = objectRdo.ToHydratedDto<T>();
-
-			switch (depthLevel)
+            
+		    IEnumerable<PropertyInfo> singleObjectFields = typeof(T).GetPublicProperties().Where(x => x.GetCustomAttribute<RelativityObjectFieldAttribute>() != null && x.GetCustomAttribute<RelativityObjectFieldAttribute>().FieldType == RdoFieldType.SingleObject);
```

Comment:
Use null propagation:
`typeof(T).GetPublicProperties().Where(x => x.GetCustomAttribute<RelativityObjectFieldAttribute>()?.FieldType == RdoFieldType.SingleObject)`

Reply:
got rid of this based on comment below